File: app/src/main/java/com/babestudios/companieshouse/data/model/company/CompanyLinks.java

package com.babestudios.companieshouse.data.model.company;

import com.babestudios.companieshouse.data.model.search.Links;
import com.google.gson.annotations.SerializedName;

public class CompanyLinks extends Links {

	@SerializedName("charges")
	public String charges;

	@SerializedName("filing_history")
	public String filingHistory;

	@SerializedName("insolvency")
	public String insolvency;

	@SerializedName("officers")
	public String officers;

	@SerializedName("persons_with_significant_control")
	public String personsWithSignificantControl;

	@SerializedName("persons_with_significant_control_statements")
	public String personsWithSignificantControlStatements;

	@SerializedName("registers")
	public String registers;

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/company/CompanyLinks.kt
package com.babestudios.companyinfouk.data.model.company

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class CompanyLinks(

		@SerializedName("charges")
		var charges: String? = null,

		@SerializedName("filing_history")
		var filingHistory: String? = null,

		@SerializedName("insolvency")
		var insolvency: String? = null,

		@SerializedName("officers")
		var officers: String? = null,

		@SerializedName("persons_with_significant_control")
		var personsWithSignificantControl: String? = null,

		@SerializedName("persons_with_significant_control_statements")
		var personsWithSignificantControlStatements: String? = null,

		@SerializedName("registers")
		var registers: String? = null,

		@SerializedName("self")
		var self: String? = null

) : Parcelable
